examples/object_detection/android-jetpack-compose/app/src/androidTest/java/com/example/objectdetection/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.objectdetection
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```

Comment:
We can delete this if it's not doing anything

Reply:
Yeah I just noticed I have the default test files and removed them